Tests/Armonik.Windows.Tests/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
```

Comment:
Is this file really necessary or only related to your machine ?

Reply:
Tests removed